### ISSUE TITLE
Increase button contrast on main webiste and app website

### ISSUE THREAD

**Issue by rafaVls** [supabase/supabase#9383]
# Feature request

## Is your feature request related to a problem? Please describe.

The text on the "accent" buttons on these two websites don't have sufficient contrast with their background, and do not meet WCAG standards for accessible text.

![Supabase website's landing page, showing firefox' developer tools analysis on the accent buttons, which have a contrast of 2.62](https://user-images.githubusercontent.com/70592708/194485596-ab655029-82f6-417b-b98c-591a3f9d43cd.png)

## Describe the solution you'd like

Changing the white text to a much darker/black text would increase readability and comply with WCAG guidelines.

![Supabase website's landing page, showing firefox' developer tools analysis on the accent buttons with text changed to black, which have a contrast of 6.33 ](https://user-images.githubusercontent.com/70592708/194486782-190662db-50ec-45b1-8f4f-b9b3b68513cc.png)
![image](https://user-images.githubusercontent.com/70592708/194487032-f67fe5bd-5161-45fe-8df8-ee1faba77b4a.png)

## Describe alternatives you've considered

Using background color works too, and keeps consistency with overall theme.

## Additional context

This problem was already addressed in #329 and apparently fixed by #435, but the changes must've been reversed at some point.

**Comment by MildTomato** [comment]
Looking at this now

**Comment by MildTomato** [comment]
Hey @rafaVls 

I think the solution that would be better here is that we look at adjusting the actual colors used rather than quick fixes like drop shadows, which, (and this is just my opinion) don't look great.

The issue is greater than just a button, what if we used a block of green with small white text on it, the contrast issue remains and it would be impractical to have a dropshadow on all the text just to try and superficially heighten the contrast rating.

**Comment by rafaVls** [comment]
Agreed, I think the solution is to change the actual text color on these elements

**Comment by rafaVls** [comment]
I can't seem to be able to run the website's developer server. I cd into apps/www, run `npm i` and `npm run dev` I get the following error:

```bash
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Loaded env from /home/rafavls/Contributing/supabase/apps/www/.env
info  - automatically enabled Fast Refresh for 1 custom loader
error - ../../node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[1]!../../node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[2]!./styles/index.css
Error: Cannot find module 'config/postcss.config'
Require stack:
- /home/rafavls/Contributing/supabase/apps/www/postcss.config.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/lib/find-config.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/build/webpack/config/blocks/css/index.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/build/webpack/config/index.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/build/webpack-config.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/server/dev/hot-reloader.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/server/dev/next-dev-server.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/server/next.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/server/lib/start-server.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/cli/next-dev.js
- /home/rafavls/Contributing/supabase/node_modules/next/dist/bin/next
```

Any clue on how to solve this?

**Comment by MildTomato** [comment]
Sorry i think you misread, I'm advocating for changing the main green background color, not the text color.

**Comment by MildTomato** [comment]
@rafaVls you need to cd to the root of the repo, then run `npm i` and `npm run dev`.
we use turbo repo now 🚀

**Comment by rafaVls** [comment]
> Sorry i think you misread, I'm advocating for changing the main green background color, not the text color.

Ah! I see. Is there a specific color you had in mind?

**Comment by MildTomato** [comment]
Update - we've been investigating on a better theming setup with perhaps CSS variables, so apologies for this getting forgotten about.

I'll have a look at this now and maybe just do some slight adjustments so we at least pass contrast tests - without making any changes that are significant. 

Going forward, we are exploring some new designs in figma, so our color palette across the whole org could be updated, and we can try to check all color ratios as we go to make sure we don't have these contrast issues come up again.

I'll follow this up shortly

**Comment by MildTomato** [comment]
This seems like an OK fix for now. I'll PR these changes now.

<img width="790" alt="Screenshot 2023-01-20 at 2 05 03 PM" src="https://user-images.githubusercontent.com/8291514/213701613-79f84b2f-53e8-4a35-8fdf-ecd5a7333d38.png">
<img width="891" alt="Screenshot 2023-01-20 at 2 05 27 PM" src="https://user-images.githubusercontent.com/8291514/213701675-4f213c98-be95-45df-8bb3-6e915b451c04.png">
